app/controllers/users_controller.rb

```diff
@@ -55,6 +56,7 @@ def show
   def update
     @user = current_user
 
+    @user.name = params[:user][:name].to_s.strip
```

Comment:
umm, maybe we don't want to add the name attribute to user model, but just use the extrauserparams